ereuse_devicehub/resources/action/views.py

```diff
@@ -12,7 +12,7 @@
     InitTransfer
 from ereuse_devicehub.resources.action.rate.v1_0 import CannotRate
 from ereuse_devicehub.resources.device.models import Component, Computer
-from ereuse_devicehub.resources.enums import SnapshotSoftware
+from ereuse_devicehub.resources.enums import SnapshotSoftware, Severity
```

Comment:
How is the severity related with the info commit?

Reply:
It is not directly related, I made a mistake with the pull request, the HID commit would have to be in https://github.com/eReuse/devicehub-teal/pull/32